tests/test_models.py

```diff
@@ -36,42 +45,168 @@ def technology_set():
     return [nuclear, natural_gas]
 
 
+@pytest.fixture
+def technology_set_2():
+    """
+    This fixture creates technologies from
+    the :class:`ThermalTechnology` subclass.
+    """
+    nuclear = ThermalTechnology(technology_name='Nuclear',
+                                capacity=5,
+                                capital_cost=6,
+                                om_cost_variable=20,
+                                om_cost_fixed=50,
+                                fuel_cost=5,
+                                ramp_up_rate=0.0,
+                                ramp_down_rate=0.0,
```

Comment:
If these values are `0.0`, does that mean the power doesn't change?

Reply:
Correct. If the ramp rates are 0.0 then the power cannot change between time periods. The model will pick some power level and it won't change throughout the model.